iOS/CFC_Tracker/GoogleOpenSource.framework/GoogleOpenSource

```diff
@@ -0,0 +1 @@
+Versions/Current/GoogleOpenSource
```

Comment:
The pull request is good enough.
Basically, a year or two from now, when people want to understand this commit, they need to have a place to look. The best option would be the commit message, since that is stored in git and moves with the files if/when they are migrated.

But since it is too late for that, let's put it in the pull request for now.